src/pudl/transform/eia861.py

```diff
@@ -1288,6 +1252,148 @@ def distributed_generation(tfr_dfs):
         dict: A dictionary of transformed EIA 861 dataframes, keyed by table name.
 
     """
+    idx_cols = [
+        'utility_id_eia',
+        'state',
+        'report_date',
+    ]
+
+    misc_cols = [
+        'backup_capacity_mw',
+        'backup_capacity_pct',
+        'distributed_generation_owned_capacity_mw',
+        'distributed_generation_owned_capacity_pct',
+        'estimated_or_actual_capacity_data',
+        'generators_number',
+        'generators_num_less_1_mw',
+        'total_capacity_mw',
+        'total_capacity_less_1_mw',
+        'utility_name_eia',
+    ]
+
+    tech_cols = [
+        'all_storage_capacity_mw',
+        'combustion_turbine_capacity_mw',
+        'combustion_turbine_capacity_pct',
+        'estimated_or_actual_tech_data',
+        'hydro_capacity_mw',
+        'hydro_capacity_pct',
+        'internal_combustion_capacity_mw',
+        'internal_combustion_capacity_pct',
+        'other_capacity_mw',
+        'other_capacity_pct',
+        'pv_capacity_mw',
+        'steam_capacity_mw',
+        'steam_capacity_pct',
+        'total_capacity_mw',
+        'wind_capacity_mw',
+        'wind_capacity_pct',
+    ]
+
+    fuel_cols = [
+        'oil_fuel_pct',
+        'estimated_or_actual_fuel_data',
+        'gas_fuel_pct',
+        'other_fuel_pct',
+        'renewable_fuel_pct',
+        'water_fuel_pct',
+        'wind_fuel_pct',
+        'wood_fuel_pct',
+    ]
+
+    # Pre-tidy transform: set estimated or actual A/E values to 'Acutal'/'Estimated'
+    raw_dg = (
+        tfr_dfs['distributed_generation_eia861'].copy()
+        .assign(
+            estimated_or_actual_capacity_data=lambda x: (
+                x.estimated_or_actual_capacity_data.map(pc.ESTIMATED_OR_ACTUAL)),
+            estimated_or_actual_fuel_data=lambda x: (
+                x.estimated_or_actual_fuel_data.map(pc.ESTIMATED_OR_ACTUAL)),
+            estimated_or_actual_tech_data=lambda x: (
+                x.estimated_or_actual_tech_data.map(pc.ESTIMATED_OR_ACTUAL))
+        )
+    )
+
+    # Split into three tables: Capacity/tech-related, fuel-related, and misc.
+    raw_dg_tech = raw_dg[idx_cols + tech_cols].copy()
+    raw_dg_fuel = raw_dg[idx_cols + fuel_cols].copy()
+    raw_dg_misc = raw_dg[idx_cols + misc_cols].copy()
+
+    ###########################################################################
+    # Transform Values:
+    # * Turn pct values into mw values
+    # * Remove old pct cols and totals cols
+    ###########################################################################
+
+    # Separate datasets into years with only pct values (pre-2010) and years with only mw values (post-2010)
```

Comment:
can you add a brief addtional comment here noting why. Like "some the pre-2010 columns are reported as percentages while the post-2010 columns are reported as MWs, so we need to standardize them"